build.sh

```diff
@@ -2,14 +2,18 @@
 
 usage()
 {
-    echo "Usage: $0 [BuildArch] [BuildType] [clean] [verbose] [platform]"
+    echo "Usage: $0 [managed] [native] [BuildArch] [BuildType] [clean] [verbose] [clangx.y] [platform] [cross]"
+    echo "managed - optional argument to build the managed code"
+    echo "native - optional argument to build the native code"
     echo "The following arguments affect native builds only:"
```

Comment:
Echoing comments from @iamjasonp -- we have no native parts to build, so this usage statement is wrong.  I know we're trying to keep these in sync, but we need to keep the -native switch and labels out of our scripts.

@StephenBonikowsky -- have you tried builds that specify these optional arguments like -cross?  We need to verify the new ones being added don't cause build failures.